seeds/utils.js

```diff
@@ -26,15 +27,3 @@ exports.splitByProp = (prop, seed, existing, differ = propsDiffer, merge = true)
     },
     { toCreate: [], toUpdate: [] },
   );
-
-const sleep = (exports.sleep = (seconds = 0.75) => {
-  const ms = seconds * 1000;
-  const start = Date.now();
-  while (Date.now() - start < ms) {}
-});
-
-exports.sleepAsync = seconds =>
```

Comment:
So we use the discourse-node-api library in the dispute-tools, where we don't need/want to have a 0.75 second delay before calls to discourse are made every single time. I think we'll need to add a more sophisticated strategy for dealing with the rate limit from within the discourse-node-api or else continue to deal with it crudely from the outside.

Reply:
Ah, I was intending to write it so that the sleep time was modifiable, but decided it could wait until later. I suppose 'later' is 'now'!

Reply:
Done!